### ISSUE TITLE
Better track errors in the exports

### ISSUE THREAD

**Issue by teknofire** [gina-alaska/imiq-map#47]
We need to email users when there are errors with exports being created.  

This will require some changes in how we gather the data from the api to build the export zip bundle.  If the api fails it doesn't check the http status code to make sure it got valid data or not.

**Comment by teknofire** [comment]
Need to capture two sets of information for this into a new model, probably named `ExportLog`

Things that can be exposed publicly and other log information that is private.

So possibly if an exception occurs, to just add a generate message about the an error occured.

If the api failed to return data a message about that.

Then private information would include the full exception error and the backtrace of the error for admin users to see.

The date the exception happened and what export was being run that had the exception (belongs_to relation on the ExportLog)

**Comment by grimm** [comment]
Do we want to use a logging gem like public activity here?

**Comment by teknofire** [comment]
No, the stuff I'm wanting to do for this is closer to what we implemented for the sandy processing.

Basically capturing all the output from the various export workers and being able to persist them to a database so an admin can view what went wrong without having to login to a server and dig through log files.  

Part of this might could probably just be handled by being able to integrate some error tracking service like papertrail or what not, but we don't have funding for that.  

I'm think this should be back burnered for now until after we have the discussion about future funding for the project.
